app/services/problem_check/missing_translator_api_key.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class ProblemCheck::MissingTranslatorApiKey < ProblemCheck
+  self.priority = "high"
+
+  def call
+    return no_problem unless SiteSetting.translator_enabled
+    name = api_key_site_setting_name
+    return no_problem if name.nil?
+    return problem if SiteSetting.get(name).blank?
+
+    no_problem
```

Comment:
Generally try to get the "failure path" to the bottom and stack all the happy paths up top. 🙂 

```suggestion
    return no_problem if SiteSetting.get(name).present?

    problem
```